maxim/logger/litellm/tracer.py

```
@@ -59,6 +68,30 @@ def __get_container_from_metadata(
             trace_id=str(uuid4()), logger=self.logger, trace_name="LiteLLM"
         )
 
+    def _extract_input_from_messages(self, messages: Any) -> Optional[str]:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `messages`

(ANN401)